src/sagemaker/lineage/context.py

```diff
@@ -283,6 +283,46 @@ def models(self) -> List[association.Association]:
         ]
         return model_list
 
+    def models_v2(
+        self, direction: LineageQueryDirectionEnum = LineageQueryDirectionEnum.DESCENDANTS
+    ) -> List[Artifact]:
+        """Get artifacts representing models from the context lineage by querying lineage data.
+
+        Returns:
+            list of Artifacts: Artifacts representing a model.
+        """
+        # Firstly query out the model_deployment vertices
+        query_filter = LineageFilter(
+            entities=[LineageEntityEnum.ACTION], sources=[LineageSourceEnum.MODEL_DEPLOYMENT]
+        )
+        model_deployment_query_result = LineageQuery(self.sagemaker_session).query(
+            start_arns=[self.context_arn],
+            query_filter=query_filter,
+            direction=direction,
+            include_edges=False,
+        )
+        model_deployment_vertices: [] = model_deployment_query_result.vertices
```

Comment:
Can `model_deployment_query_result.vertices` be None?
```
>>> m = [1,2,3]
>>> n = None
>>> o: [] = m
>>> o
[1, 2, 3]
>>> o: [] = n
>>> print(o)
None
```

Reply:
> Can `model_deployment_query_result.vertices` be None?
> 
> ```
> >>> m = [1,2,3]
> >>> n = None
> >>> o: [] = m
> >>> o
> [1, 2, 3]
> >>> o: [] = n
> >>> print(o)
> None
> ```

Fixed